### DISCUSSION TITLE
install the gazebo9 and superbuild problem

### DISCUSSION THREAD

**Discussion by jimingda** [robotology/community#650] [Q&A] [answered]
Hi,

I tried to use the gazebo sim garden and installed it, and then i found the default gazebo is unavailable.
Then i uninstall the gazebo sim garden, and install the default gazebo9 `sudo apt install gazebo9`
After which i rebuild the robotology-superbuild, during the make process got the following error.

Thanks.
```
[ 56%] Completed 'icub-models'
[ 59%] Built target icub-models
[ 61%] Skipping update step for 'GazeboYARPPlugins'
[ 62%] Performing configure step for 'GazeboYARPPlugins'
Not searching for unused variables given on the command line.
loading initial cache file /usr/local/src/robot/robotology-superbuild/build/src/GazeboYARPPlugins/CMakeFiles/YCMTmp/GazeboYARPPlugins-cache-Release.cmake
-- Found YARP: /usr/local/src/robot/robotology-superbuild/build/install/lib/cmake/YARP (found version "3.7.0")
CMake Error at CMakeLists.txt:50 (find_package):
  By not providing "FindGazebo.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Gazebo", but
  CMake did not find one.

  Could not find a package configuration file provided by "Gazebo" with any
  of the following names:

    GazeboConfig.cmake
    gazebo-config.cmake

  Add the installation prefix of "Gazebo" to CMAKE_PREFIX_PATH or set
  "Gazebo_DIR" to a directory containing one of the above files.  If "Gazebo"
  provides a separate development package or SDK, be sure it has been
  installed.
```

**Comment by traversaro** [comment, marked as answer]
Which Ubuntu distribution are you using? To install Gazebo Classic on your Ubuntu distribution using apt dependencies, see the instructions in https://github.com/robotology/robotology-superbuild#robotology_uses_gazebo . If you do not want to use Gazebo Classic related software, you can set the CMake option of the superbuild `ROBOTOLOGY_USES_GAZEBO` to `OFF`.

**Reply by jimingda** [reply]
Thanks.
My ubuntu is `"20.04.4 LTS (Focal Fossa)"`
After follow the instruction, the superbuild can make correctly.
```
sudo apt-get install gazebo11
sudo apt-get install libgazebo11-dev
```
